builtin/files/userdata/cloud-config-controller

```diff
@@ -3449,6 +3452,27 @@ write_files:
           {{range $f := .APIServerFlags}}
           - --{{$f.Name}}={{$f.Value}}
           {{ end -}}
+          {{- if .Kubernetes.KubeApiServer.ComputeResources -}}
+          resources:
+            {{- if .Kubernetes.KubeApiServer.ComputeResources.Requests -}}
+            requests:
+              {{- if .Kubernetes.KubeApiServer.ComputeResources.Requests.Cpu -}}
+              cpu: {{.Kubernetes.KubeApiServer.ComputeResources.Requests.Cpu }}
+              {{- end }}
+              {{- if .Kubernetes.KubeApiServer.ComputeResources.Requests.Memory -}}
+              memory: {{.Kubernetes.KubeApiServer.ComputeResources.Requests.Memory }}
+              {{- end }}
+            {{- end }}
+            {{- if .Kubernetes.KubeApiServer.ComputeResources.Limits -}}
+            limits:
+              {{- if .Kubernetes.KubeApiServer.ComputeResources.Limits.Cpu -}}
+              cpu: {{.Kubernetes.KubeApiServer.ComputeResources.Limits.Cpu }}
+              {{- end }}
+              {{- if .Kubernetes.KubeApiServer.ComputeResources.Limits.Memory -}}
+              memory: {{.Kubernetes.KubeApiServer.ComputeResources.Limits.Memory }}
+              {{- end }}
```

Comment:
I'm not sure we should limit the apiserver memory by default - this could result in OOMs on larger clusters where defaults are too small.

Reply:
Limits shouldn't be enabled by default, they're uncommented in the template and not in the `NewCluster` function.

Reply:
👍

Reply:
👍